src/algorithms/eigsolvers/Arnoldi.m

```diff
@@ -24,39 +24,67 @@
             end
         end
         
-        function [V, D, flag] = eigsolve(alg, A, v, howmany, sigma)
+        function varargout = eigsolve(alg, A, v0, howmany, sigma)
```

Comment:
Maybe a quick docstring for this function, `varargout` is not very helpful in remembering what it returns in what order, depending on `nargout`

Reply:
Done.